hack/generate/csv.sh

```diff
@@ -88,6 +88,7 @@ kafka_image "kafka-ch-controller__controller"      "${eventing_kafka}-consolidat
 kafka_image "DISPATCHER_IMAGE"                     "${eventing_kafka}-consolidated-dispatcher"
 kafka_image "kafka-ch-dispatcher__dispatcher"      "${eventing_kafka}-consolidated-dispatcher"
 kafka_image "kafka-webhook__kafka-webhook"         "${eventing_kafka}-webhook"
+kafka_image "storage-version-migration-kafka-kafka-$(metadata.get dependencies.eventing_kafka)__migrate" "${eventing_kafka}-storage-version-migration"
```

Comment:
I used `kafka-kafka` here, b/c we have `eventing-eventing` for "eventing core"...